core/server/src/test/java/alluxio/worker/block/BlockMasterClientTest.java

```diff
@@ -49,8 +49,8 @@ public void unsupportedVersionTest() throws Exception {
 
     BlockMasterClient client =
         new BlockMasterClient(NetworkAddressUtils.getConnectAddress(
-            NetworkAddressUtils.ServiceType.MASTER_RPC, WorkerContext.getConf()),
-            WorkerContext.getConf());
+        NetworkAddressUtils.ServiceType.MASTER_RPC, WorkerContext.getConf()),
```

Comment:
We usually use a continuation indentation in cases like this, as recommended by the [google style formatter](http://alluxio.org/documentation/master/resources/alluxio-code-formatter-eclipse.xml). If checkstyle doesn't support checking the indentation style we want, we won't be able to enable forceStrictCondition.